react-example/cypress/integration/inapp/inapp.spec.ts

```diff
@@ -5,7 +5,7 @@ describe('Requesting In-App Messages', () => {
     cy.intercept(
       {
         method: 'GET',
-        url: '/api/inApp/getMessages*'
+        url: '/api/inApp/web/getMessages*'
```

Comment:
can we turn this into a variable? that way, if it ever changes again we don't need to update it in multiple places :)

Reply:
in fact, we could probably make it some global const and use it in all of these places.

Reply:
Great suggestion, updated the PR!